src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+
+  if (days < 3) {
+    return pricePerDay * days;
+  } else if (days >= 3 && days <= 6) {
+    return pricePerDay * days - 20;
```

Comment:
The condition `days >= 3 && days <= 6` can be simplified to `days <= 6` because the previous `if` statement already returns for `days < 3`. Also, replace the hard-coded value '20' with a named constant.